django/shortcuts.py

```diff
@@ -13,6 +13,7 @@
 from django.template import loader
 from django.urls import NoReverseMatch, reverse
 from django.utils.functional import Promise
+from django.utils.translation import gettext_lazy as _
```

Comment:
Why gettext_lazy rather than gettext? I usually only see lazy at the class level.

Reply:
> If you happen to have an example handy, that would be great to see.

```
django/contrib/admin/filters.py
django/db/models/base.py
django/forms/boundfield.py
```

Reply:
Cool, having those examples will help me find a better shorthand to use than "class-level". :-)

My understanding for BoundField: the instances get assigned as class attributes, so it's class-level, not request-level functionality. The admin example is probably the same thing. The models example looks like some kind of dictionary default, so not really anything to do with classes but more just a timing thing, in that a default could have been set well before transactions were activated? Something like that :-)